saleor/graphql/attribute/constants.py

```diff
@@ -0,0 +1,26 @@
+from ...permission.enums import (
+    PagePermissions,
+    ProductPermissions,
+    ProductTypePermissions,
+)
+from .enums import AttributeTypeEnum
+
+PERMISSIONS_MAP: dict = {
+    AttributeTypeEnum.PRODUCT_TYPE.value: {
+        "default": (
+            ProductPermissions.MANAGE_PRODUCTS,
+            ProductTypePermissions.MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES,
+        )
+    },
+    AttributeTypeEnum.PAGE_TYPE.value: {
+        "default": (
+            PagePermissions.MANAGE_PAGES,
+            ProductTypePermissions.MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES,
+        ),
+        "create": (
+            ProductPermissions.MANAGE_PRODUCTS,
+            PagePermissions.MANAGE_PAGES,
+            ProductTypePermissions.MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES,
```

Comment:
For pages it should be `MANAGE_PAGE_TYPES_AND_ATTRIBUTES` instead of `MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES`. Also, are you sure that `MANAGE_PRODUCTS` is correct here